tietokanta/src/main/resources/db/migration/V1_960__.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE geometriapaivitys
+    ADD COLUMN seuraava_paivitys TIMESTAMP,
+    ADD COLUMN edellinen_paivitysyritys TIMESTAMP,
+    ADD COLUMN paikallinen BOOLEAN DEFAULT FALSE,
+    ADD COLUMN kaytossa BOOLEAN DEFAULT TRUE;
+
+COMMENT ON COLUMN geometriapaivitys.paikallinen IS 'Kun et halua ladata aineistoa palvelimelta vaan käyttää manuaalisesti tiedostosijaintiin vietyä shapefileä, aseta tähän arvo TRUE.';
```

Comment:
Varmistan vielä tässä: Miten ja missä tilanteissa tätä sitten käytetään? Käydäänkö tämä puukottamassa suoraan kantaan siinä ympäristössä, missä näitä halutaan venkslata?

Reply:
Joo, kannan kautta säädetään.